### DISCUSSION TITLE
Getting started with remark-attributes

### DISCUSSION THREAD

**Discussion by etdr** [manuelmeister/remark-attributes#3] [General]
Hey, just found this last week after starting to write my own version of this and am considering helping out, could I use this as a thread to ask questions or is there some other way you'd prefer to communicate?

**Comment by manuelmeister** [comment]
Cool! Thanks.
Yes. I just opened the `Discussions` tab. I think I would prefer to use the `Issues` tab for action items / todos and keep open end discussion here.

May I ask, why do you need this plugin? Did you use markdown-it-attrs before?

**Comment by etdr** [comment]
That's perfect, sure. I actually am just now building a blog site with SvelteKit and Mdsvex (MDX for Svelte) and for some reason thought this would be more standard syntax for classes, ids, and other attributes but I guess it isn't, so I'm starting with this. My main thinking is that I'd like to be able to position images using classes within Markdown, if that makes sense.
